packaging/os/zypper.py

```diff
@@ -325,24 +341,56 @@ def package_absent(m, name):
 
     return retvals
 
+
+def repo_refresh(m):
+    "update the repositories"
+    retvals = {'rc': 0, 'stdout': '', 'stderr': '', 'changed': False, 'failed': False}
+
+    cmd = get_cmd(m, 'refresh')
+
+    retvals['cmd'] = cmd
+    result, retvals['rc'], retvals['stdout'], retvals['stderr'] = parse_zypper_xml(m, cmd)
+    if retvals['rc'] == 0:
+        # refresh was successed
+        retvals['changed'] = True
```

Comment:
Can we somehow detect if zypper fetched new information? And only set changed = True in this case? This would prevent changed entries on every ansible run (otherwise making it less useful to catch config drift).